CMakeLists.txt

```diff
@@ -205,6 +205,13 @@ endif ()
 
 set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
 
+#################################################################################
+# INSTALL
+#################################################################################
+
+install (TARGETS ${PROJECT_NAME} DESTINATION lib)
+install (FILES "${BS_DIR_INC}/common/instrumentation/api/metrics_discovery_api.h" DESTINATION include)
```

Comment:
The header should probably land in the include/mdapi rather than just include. Assuming default destination is /usr. In /usr/include system headers are placed and this one is not the one.

Reply:
dvrogozh, I made a change to the PR, can you review?